code/sys/sys_unix.c

```diff
@@ -552,7 +553,10 @@ void Sys_Sleep( int msec )
 		if( msec < 0 )
 			msec = 10;
 
-		usleep( msec * 1000 );
+		struct timespec req;
```

Comment:
Can you move the variable declaration to the beginning of the scope? The line after { a few lines up.

Reply:
Done.